pages/pipelines/artifacts.md.erb

```diff
@@ -27,60 +70,91 @@ steps:
 ```
 {: codeblock-file="pipeline.yml"}
 
-## Uploading artifacts from scripts
+## Use the Buildkite agent to upload artifacts
+
+Within a build, run the `buildkite-agent artifact upload` command to upload artifacts from a script.
+The agent's `upload` command arguments are one or more file paths and [glob patterns](/docs/agent/v3/cli-artifact#uploading-artifacts-artifact-upload-glob-syntax).
 
-Upload artifacts from your build scripts with the `buildkite-agent artifact` command.
-The following example uploads `build.tar.gz` from the `pkg` directory to `pkg/build.tar.gz`:
+The following example uploads a `build.tar.gz` file from the `pkg` directory:
 
-```bash
+```shell
 buildkite-agent artifact upload pkg/build.tar.gz
 ```
 
-For full documentation of the `buildkite-agent artifact upload` command see the [buildkite-agent artifact upload](/docs/agent/v3/cli-artifact#uploading-artifacts) documentation.
-
+The `buildkite-agent artifact upload` command supports several options and environment variables.
+For complete usage instructions, read the [`buildkite-agent artifact upload`](/docs/agent/v3/cli-artifact#uploading-artifacts) documentation.
 
 ## Downloading artifacts
 
-You can download artifacts created by a build job, from inside a running build, using the `buildkite-agent artifact download` command. For example, to download all artifacts in the `logs` directory (including subdirectories, which will be recreated) to `local-logs/logs/`:
+When you upload an artifact, Buildkite records the file's contents, the complete path the file was uploaded from, and details of the build step it originated from, so you can retrieve artifacts by name, path, or build.
+
+You can download artifacts by running the `buildkite-agent artifact download` command during a build or by making a request to the artifacts REST API.
+
+## Use the Buildkite agent to download artifacts
+
+Within a build, run the `buildkite-agent artifact download` command to download artifacts from a script.
+The agent's `download` command arguments are a file path or [glob pattern](/docs/agent/v3/cli-artifact#uploading-artifacts-artifact-upload-glob-syntax) and a destination path.
+
+The `buildkite-agent artifact download` command supports several options and environment variables.
+For complete usage instructions, read the [`buildkite-agent artifact download`](/docs/agent/v3/cli-artifact#downloading-artifacts) documentation.
+
+### Example: download one artifact
+<!-- TODO: Find out if this is a conventional way to head a series of examples -->
 
-```bash
-buildkite-agent artifact download logs/* local-logs/
+The following example downloads a `pkg/build.tar.gz` artifact to the `archives` directory:
```

Comment:
Strictly speaking, it could be anywhere. `buildkite-agent artifact download`, despite the name, is not that tightly coupled to the work of an agent; it's more of a thin wrapper over the REST API. Given a PAT, you could run `buildkite-agent artifact download` with some options to download arbitrary artifacts from any build. (I think I know _why_ `buildkite-agent` is both a job runner and a CLI to the API, but it makes things subtly messy to explain.)

That said, I can see where your coming from here. I fictionalized some context for this case, but I've carefully not said that this is the only context in which the command could run. b8b3dc8b